tpm2tools/template.go

```diff
@@ -47,6 +47,20 @@ func defaultRSADecrypt() *tpm2.RSAParams {
 	}
 }
 
+func defaultECCDecrypt() *tpm2.ECCParams {
```

Comment:
Calling them `default*Decrypt` was a bit confusing on my part `defaultRSAParams` and `defaultECCParams` are better names, so I'd change them.

Reply:
Done